File: C/1010.c
#include <stdio.h>
 
int main() {
 
    int id1, number1, id2, number2;
    float value1, value2;

    scanf("%d %d %f", &id1, &number1, &value1);
    scanf("%d %d %f", &id2, &number2, &value2);

    printf("VALOR A PAGAR: R$ %.2f\n", (number1 * value1 + number2 * value2));
 
    return 0;
}

File: Javascript/1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let value1 = lines.shift().split(' ');
let value2 = lines.shift().split(' ');
let totalvalue = Number(value1[1]) * Number(value1[2]) + Number(value2[1]) * Number(value2[2]); 
console.log("VALOR A PAGAR: R$ " + totalvalue.toFixed(2));